src/components/Select/index.scss

```diff
@@ -39,11 +39,6 @@
       }
     }
   }
-  &__menu-list {
-    &:hover {
-      background-color: $base-white;
```

Comment:
i added these lines in [this PR](https://github.com/AccelByte/justice-ui-library/pull/57/files#diff-871d1b8aceb6512a86325d6dd96d3355a1ea12d2a116a47c81517ec3a4fe53e7R41) because if a select is inside .styled-atlaskit-form, menu-list's background will be gray if it is hovered.

i wasn't aware that by adding this, it causes another problem in which if a select has many options and we try to scroll it, the option doesn't show up until we unhover the select.

Reply:
next action: adjust .styled-atlaskit-form in AP